src/lms-ds-loader/edfi_lms_ds_loader/mssql_lms_operations.py

```diff
@@ -210,3 +215,15 @@ def soft_delete_from_production(self, table: str, sourceSystem: str) -> None:
 
         rowcount = self._exec(statement)
         logger.debug(f"Soft-deleted {rowcount} records in table `{table}`")
+
+    def get_processed_files(self, resource_name: str) -> pd.DataFrame:
+        query = f"""
+SELECT FullPath FROM lms.ProcessedFiles
+WHERE ResourceName = '{resource_name}'""".strip()
+        return self._query(query)
+
+    def add_processed_file(self, path: str, resource_name: str, rows: int):
+        statement = f"""
+INSERT INTO lms.ProcessedFiles(FullPath, ResourceName, NumberOfRows)
+VALUES ('{path}','{resource_name}', {rows})""".strip()
+        return self._exec(statement)
```

Comment:
I believe this code is safe from SQL injection attacks since we're not taking in any user input. However, it would be better to use [SQL Alchemy's parameter injection method](https://security.openstack.org/guidelines/dg_parameterize-database-queries.html) instead of directly putting the string values into the statement as was done here.